bin/pycbc_bank_verification

```diff
@@ -36,6 +36,7 @@ import os, sys
 import copy
 import logging
 import numpy
+import h5py
 from glue.ligolw import ligolw, table, lsctables, utils as ligolw_utils
```

Comment:
For now I still want to support XML banks. So I'll keep these in this patch, but dropping XML support is also something that has been discussed in an sbank context. So maybe post-O2 we can look at removing XML bank support globally. (Previously gstlal required some meta-information in xml files, that may not have made it to HDF yet, so I'd like to make deprecating XML a gentle process).